bin/send_email.py

```diff
@@ -143,31 +145,41 @@ def send_email(view_id: str, submission_id: str, email_with_score: str):
     # Get the Synapse users to send an e-mail to
     ids_to_notify = helpers.get_participant_id(syn, submission_id)
 
-    # Create the subject and body of the e-mail message, depending on submission status
-    subject = (
-        f"Evaluation Success: {submission_id}"
-        if submission_annotations.status == "VALIDATED"
-        else f"Evaluation Failed: {submission_id}"
-    )
-    body = email_template(
-        submission_annotations.status,
-        email_with_score,
-        submission_id,
-        view_id,
-        submission_annotations.score,
-        submission_annotations.reason,
-    )
+    # Create the subject and body of the e-mail message, depending on
+    # the notification type and submission status:
+    if notification_type.upper() == "BEFORE":
+        # Before-evaluation notification
+        subject = f"Evaluation Started: {submission_id}"
+        body = (
+            f"Dear participant,\n\n"
+            f"Your submission **{submission_id}** is now being evaluated. "
```

Comment:
I believe the default content type for emails sent by Synapse are `text/HTML` so this should be using the `<b>`/`</b>` tags, rather than Markdown's `**`.

Reply:
Good to know for future styling 👀 Updated, thanks!

<img width="621" alt="image" src="https://github.com/user-attachments/assets/8decfd76-6129-4f01-86e7-67f60a51bef0" />